analytics-core/src/main/java/com/liferay/analytics/android/Analytics.kt

```diff
@@ -31,7 +34,8 @@ import org.koin.standalone.StandAloneContext.startKoin
  * @author Igor Matos
  */
 class Analytics private constructor(
```

Comment:
In this case, we could use the kotlin object instead of a class with private constructor and a companion init method. But just a suggestion, not a change for this PR.

Reply:
Yeah. Maybe it's a good approach, but I would have to initiate all parameters of my singleton on init argument as I'm doing this today, the problem is that `lateinit` will not work for primitive types. I can try to figure out a better approach to this.
```
object Analytics {
			val endpointURL = ""
			val dataSourceId = ""
			val fileStorage = ""
			var  flushInterval: Long
		}
```